templates/fail2ban/jail.conf.j2

```diff
@@ -0,0 +1,9 @@
+[gitea]
+enabled = true
+port = http,https
+filter = gitea
+logpath = {{ gitea_home }}/log/gitea.log
+maxretry = 10
+findtime = 3600
+bantime = 900
+action = ufw
```

Comment:
can you make the action templated as well ? iirc you have multiple types of actions for this (like iptables and what not)